README.md

```diff
@@ -187,23 +187,57 @@ module.exports = {
 
 The `lexers` option let you configure which Lexer to use for which extension. Here is the default:
 
-Note the presence of a `default` which will catch any extension that is not listed. There are 3 lexers available: `HandlebarsLexer`, `HTMLLexer` and `JavascriptLexer`. Each has configurations of its own. If you need to change the defaults, you can do it like so:
+Note the presence of a `default` which will catch any extension that is not listed.
+There are 5 lexers available: `HandlebarsLexer`, `HTMLLexer`, `JavascriptLexer`,
+`JsxLexer`, and `TypescriptLexer`. Each has configurations of its own.
+If you need to change the defaults, you can do it like so:
+
+#### Javascript
+The Javascript lexer uses [Acorn](https://github.com/acornjs/acorn) to walk through your code and extract references
+translation functions. If your code uses features not supported natively by Acorn, you can enable support through
+`injectors` and `plugins` configuration. This is an example configuration that adds all non-jsx plugins supported by acorn
+at the time of writing:
```

Comment:
I would still mention that it's the responsability of the user to `npm install` or `yarn add` the dependencies. They are NOT included in the parser package.